src/test/java/build/buf/protovalidate/internal/celext/CustomOverloadTest.java

```diff
@@ -63,6 +79,58 @@ public void testIsNan() {
     }
   }
 
+  @Test
+  public void testIsNanUnsupported() {
+    List<String> testCases = ImmutableList.of("'foo'.isNan()");
+    for (String testCase : testCases) {
+      Val val = eval(testCase).getVal();
+      assertThat(Err.isError(val)).isTrue();
+      assertThatThrownBy(() -> val.convertToNative(Exception.class))
+          .isInstanceOf(UnsupportedOperationException.class);
+    }
+  }
+
+  @Test
+  public void testUnique() {
+    Map<String, Boolean> testCases =
+        ImmutableMap.<String, Boolean>builder()
+            .put("[].unique()", true)
+            .put("[true].unique()", true)
+            .put("[true, false].unique()", true)
+            .put("[true, true].unique()", false)
+            .put("[1, 2, 3].unique()", true)
+            .put("[1, 2, 1].unique()", false)
+            .put("[1u, 2u, 3u].unique()", true)
+            .put("[1u, 2u, 2u].unique()", false)
+            .put("[1.0, 2.0, 3.0].unique()", true)
+            .put("[3.0,2.0,3.0].unique()", false)
+            .put("['abc', 'def'].unique()", true)
+            .put("['abc', 'abc'].unique()", false)
+            .put("[b'abc', b'123'].unique()", true)
+            .put("[b'123', b'123'].unique()", false)
+            // Previously, the unique() method returned false here as both bytes were converted
+            // to UTF-8. Since both contain invalid UTF-8, this would lead to them treated as equal
+            // because they'd have the same substitution character.
+            .put("[b'\\xFF', b'\\xFE'].unique()", true)
```

Comment:
This failed before fixing the unique implementation (this test doesn't exist in the Go implementation but it doesn't suffer the same issues).